src/1_utils.js

```diff
@@ -586,15 +586,21 @@ utils.getHostedDeepLinkData = function() {
 
 
 /**
- * @return {string} code
+ * Returns the user's preferred language
  */
 utils.getBrowserLanguageCode = function() {
 	var code;
 	try {
-		code = (navigator.language || navigator.browserLanguage || 'en').split(/[^a-z^A-Z0-9-]/).shift().toLowerCase();
+		if (navigator.languages && navigator.languages.length>0) {
+			code = navigator.languages[0];
+		}
+		else if (navigator.language) {
+			code = navigator.language;
+		}
+		code = code.substring(0, 2).toUpperCase();
 	}
 	catch (e) {
-		code = 'en';
+		code = null;
```

Comment:
Why are we changing the default language in case of exception?

Reply:
@dmitrig01 it turns out that this function is used by Journeys as well so we'd rather send back 'en' to ensure that Journeys render something readable in English in the case of an exception. 

I'd like to revert this back to 'en' if possible.

Reply:
can we leave that in the journeys-specific section? I would rather send no language than the wrong language in v2/event.

Reply:
Yup, we can do that!